include/lsst/meas/algorithms/KernelPsfFactory.h

```diff
@@ -13,6 +13,7 @@
 #include "lsst/afw/table/io/InputArchive.h"
 #include "lsst/afw/table/io/OutputArchive.h"
 #include "lsst/afw/table/io/CatalogVector.h"
+#include "lsst/afw/table/aggregates.h"
```

Comment:
That's a lot of detailed imports. If it does not cause too much build slowdown or bloat I suggest importing table.h